backend/battling/tests/test_view.py

```diff
@@ -191,10 +194,24 @@ def test_does_not_create_team_with_repeated_pokemon(self):
             "pokemon_3_position": 3,
         }
         response = self.auth_client.post(
-            reverse("create_team", kwargs={"pk": self.team.id}), pokemon_data, follow=True
+            reverse("create_team", kwargs={"pk": self.opponent_team.id}), pokemon_data, follow=True
         )
 
         self.assertEqual(
             response.context_data["form"].errors["__all__"][0],
             "You can't choose the same Pokemon more than once.",
         )
+
+    def test_run_battle_and_send_result_email(self):
```

Comment:
You need to test if the email is being sent, for that to work you need to mock the send_template_email call and assert that was called with the correct parameters.